dtests/src/resources/scripts/complexTypes/complextype.sql

```diff
@@ -0,0 +1,192 @@
+-- ####################################################################################################################################################################################################################################
+-- Create Schema -> Array Type
+CREATE SCHEMA ST;
+
+-- Create Table
+CREATE TABLE IF NOT EXISTS ST.Student(rollno Int, name String, marks ARRAY<Double>) USING column;
+
+--Insert the values (data)
+INSERT INTO ST.Student SELECT 1,'Mohit Shukla', Array(97.8,85.2,63.9,45.2,75.2,96.5);
+INSERT INTO ST.Student SELECT 2,'Nalini Gupta',Array(89.3,56.3,89.1,78.4,84.1,99.2);
+INSERT INTO ST.Student SELECT 3,'Kareena Kapoor',Array(99.9,25.3,45.8,65.8,77.9,23.1);
+INSERT INTO ST.Student SELECT 4,'Salman Khan',Array(99.9,89.2,85.3,90.2,83.9,96.1);
+INSERT INTO ST.Student SELECT 5,'Aranav Goswami',Array(90.1,80.1,70.1,60.1,50.1,40.1);
+INSERT INTO ST.Student SELECT 6,'Sudhir Chudhari',Array(81.1,81.2,81.3,81.4,81.5,81.6);
+INSERT INTO ST.Student SELECT 7,'Anjana Kashyap',Array(71.2,65.0,52.3,89.4,95.1,90.9);
+INSERT INTO ST.Student SELECT 8,'Navika Kumar',Array(95.5,75.5,55.5,29.3,27.4,50.9);
+INSERT INTO ST.Student SELECT 9,'Atul Singh',Array(40.1,42.3,46.9,47.8,44.4,42.0);
+INSERT INTO ST.Student SELECT 10,'Dheeraj Sen',Array(62.1,50.7,52.3,67.9,69.9,66.8);
+
+-- Create Views
+CREATE TEMPORARY VIEW StudentMark AS SELECT rollno,name,explode(marks) AS Marks FROM ST.Student;
+
+-- Queries
+SELECT * FROM ST.Student ORDER BY rollno;
+SELECT rollno, marks[0] AS Maths, marks[1] AS Science, marks[2] AS English, marks[3] AS Computer, marks[4] AS Music, marks[5] FROM ST.Student WHERE name = 'Salman Khan';
+SELECT rollno, name, explode(marks) as Marks FROM ST.Student;
+SELECT name,SUM(Marks) AS Total FROM StudentMark GROUP BY name ORDER BY Total DESC;
+SELECT name,MAX(Marks),MIN(Marks) FROM StudentMark GROUP BY name;
+
+-- Clean up -> Array Type
+DROP TABLE IF EXISTS ST.Student;
```

Comment:
Add drop statements before create. In case the script fails for some reason, then the re-execution will fail with table already exists exception.